### ISSUE TITLE
update python

### ISSUE THREAD

**Issue by Soheab** [codenvy/codenvy#2750]
Please update python to the latest or even 3.7 is good because now its 3.5.8.. very outdated

**Comment by thecraftman** [comment]
yes you should because python 3.7 now supports built in breakpoint() and context variables
